### PR TITLE
Add anomaly score explanation for ML get records endpoint

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/elasticsearch/pull/91177

### CODE REVIEW

File: specification/ml/_types/Anomaly.ts [hunk 1]
```diff
@@ -67,3 +71,46 @@ export class Influence {
...
+  /**
+   * Type of the detected anomaly: `spike` or `dip`.
+   */
+  anomaly_type?: string
+  /**
+   * Indicates reduction of anomaly score for the bucket with large confidence intervals. If a bucket has large confidence intervals, the score is reduced.
+   */
+  high_variance_penalty?: boolean
+  /**
+   * If the bucket contains fewer samples than expected, the score is reduced. If the bucket contains fewer samples than expected, the score is reduced.
```
Comment (valeriy42) [inline specification/ml/_types/Anomaly.ts:93]: It is the same sentence twice.

File: output/schema/schema.json [hunk 2]
```diff
@@ -121388,7 +121400,137 @@
...
+          "type": {
+            "kind": "instance_of",
+            "type": {
+              "name": "boolean",
+              "namespace": "_builtins"
+            }
+          }
+        },
+        {
+          "description": "If the bucket contains fewer samples than expected, the score is reduced. If the bucket contains fewer samples than expected, the score is reduced.",
```
Comment (valeriy42) [inline output/schema/schema.json:121461]: This is the same sentence twice.

**Review by valeriy42 — COMMENTED** [review]
In one of the comments, the same sentence is repeated twice.

File: specification/ml/_types/Anomaly.ts (hunk 1 above)
Comment (lcawl) [inline specification/ml/_types/Anomaly.ts:93]:
```suggestion
   * If the bucket contains fewer samples than expected, the score is reduced.
```

Comment (lcawl) [reply specification/ml/_types/Anomaly.ts]: Thanks for catching that!

File: output/schema/schema.json (hunk 2 above)
Comment (lcawl) [reply output/schema/schema.json:121461]: Thanks, fixed in c160143b1e5456dc4e0b9675e922ae7121177a35

**Review by valeriy42 — APPROVED** [review]
LGTM

**Pull request merged**